docs/historical_model.rst

```diff
@@ -179,6 +179,75 @@ IMPORTANT: Setting `custom_model_name` to `lambda x:f'{x}'` is not permitted.
            An error will be generated and no history model created if they are the same.
 
 
+Custom History Manager and Historical QuerySets
+-----------------------------------------------
+
+To manipulate the history manager or the historical ``QuerySet`` of
+``HistoricalRecords``, you can specify the ``history_manager`` and
+``historical_queryset`` options. The values must be subclasses
+of ``simple_history.manager.HistoryManager`` and
+``simple_history.manager.HistoricalQuerySet``, respectively.
+
+Keep in mind, you can use either or both of these options. To understand the
+difference between a ``Manager`` and a ``QuerySet``,
+see `Django's Manager documentation`_.
+
+.. code-block:: python
+    from datetime import timedelta
+    from django.db import models
+    from django.utils import timezone
+    from simple_history.manager import HistoryManager, HistoricalQuerySet
+    from simple_history.models import HistoricalRecords
+
+
+    class HistoryQuestionManager(HistoryManager):
+        def published(self):
+            return self.filter(pub_date__lte=timezone.now())
+
+
+    class HistoryQuestionQuerySet(HistoricalQuerySet):
+        def question_prefixed(self):
+            return self.filter(question__startswith="Question: ")
+
+
+    class Question(models.Model):
+        pub_date = models.DateTimeField("date published")
+        history = HistoricalRecords(
+            history_manager=HistoryQuestionManager,
+            historical_queryset=HistoryQuestionQuerySet,
+        )
+
+    # This is now possible:
+    queryset = Question.history.published().question_prefixed()
+
+
+To reuse a ``QuerySet`` from the model, see the following code example:
+
+.. code-block:: python
+    from datetime import timedelta
+    from django.db import models
+    from django.utils import timezone
+    from simple_history.models import HistoricalRecords
+    from simple_history.manager import HistoryManager, HistoricalQuerySet
+
+
+    class QuestionQuerySet(HistoricalQuerySet):
+        def question_prefixed(self):
+            return self.filter(question__startswith="Question: ")
+
+
+    class HistoryQuestionQuerySet(QuestionQuerySet, HistoricalQuerySet):
+        """Redefine ``QuerySet`` with base class ``HistoricalQuerySet``."""
+
+
+    class Question(models.Model):
+        pub_date = models.DateTimeField("date published")
+        history = HistoricalRecords(historical_queryset=HistoryQuestionQuerySet)
+        manager = models.Manager.from_queryset(QuestionQuerySet)()
```

Comment:
Could `as_manager()` make the code slightly simpler, or would it be better to keep the suggestion of using `from_queryset()`?
```suggestion
        manager = QuestionQuerySet.as_manager()
```

Reply:
I think either works fine. The original shows off a more complex feature of Django, but that's not our responsibility.